components/admin/contentPage/DestinationCards.tsx

```diff
@@ -194,6 +219,11 @@ const DestinationCards: React.FC = () => {
             variant="outlined"
             required
             sx={{ m: 1 }}
+            InputProps={{
```

Comment:
Can't you handle this in theme as given below?
 components: {
    MuiTextField: {
      styleOverrides: {
        root: {
          // Styles for the root of the TextField
          margin: '8px', // Equivalent to m: 1
        },
        input: {
          // Styles for the input element
          color: 'black',
        },
      },
    },